app/views/foreman/smart_proxies/_content_sync.html.erb

```diff
@@ -21,15 +21,33 @@
   </div>
 
   <br>
-
-  <div>
-    <a ng-show="syncState.is(syncState.SYNCING, syncState.SYNC_TRIGGERED, syncState.CANCEL_TRIGGERED)"
-      ng-click="cancelSync()" class="btn btn-default" ng-disabled="!syncState.is(syncState.SYNCING)">
+  <div ng-show="syncState.is(syncState.SYNCING, syncState.SYNC_TRIGGERED, syncState.CANCEL_TRIGGERED)">
+    <a ng-click="cancelSync()" class="btn btn-default" ng-disabled="!syncState.is(syncState.SYNCING)">
       <span translate>Cancel Sync</span>
     </a>
-    <a ng-hide="syncState.is(syncState.SYNCING, syncState.SYNC_TRIGGERED, syncState.CANCEL_TRIGGERED)"
-      ng-click="syncCapsule()" class="btn btn-default">
-      <span translate>Synchronize</span>
-    </a>
   </div>
+  <div class="dropdown" ng-hide="syncState.is(syncState.SYNCING, syncState.SYNC_TRIGGERED, syncState.CANCEL_TRIGGERED)">
+    <button class="btn btn-default dropdown-toggle" type="button" id="syncDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+      Synchronize
+      <span class="caret"></span>
+    </button>
+    <ul class="dropdown-menu" aria-labelledby="syncDropdown">
+      <li>
+        <a ng-click="syncCapsule()">
+          <span translate>Standard Sync</span>
+        </a>
+      </li>
+      <li>
+        <a ng-click="syncCapsule(true)">
+          <span translate>Force Full Sync</span>
+        </a>
+      </li>
+    </ul>
+  </div>
+  <i class="fa fa-question-circle inline-icon"
+     uib-tooltip="{{ 'Standard sync will sync only repositories that have changed, 
+                     Force full sync will sync all repositories' | translate }}"
+     tooltip-animation="false"
+     tooltip-append-to-body="true">
+  </i>
```

Comment:
@waldenraines Any ideas on how to get these to be inline? ![these](http://www.clipular.com/c/6220639821365248.png?k=3m6Jr9wZIUvwKWtMe6N6M8jxJfU)